api/turing/config/config.go

```diff
@@ -496,6 +497,33 @@ func Load(filepaths ...string) (*Config, error) {
 		return nil, fmt.Errorf("failed to unmarshal config values: %s", err)
 	}
 
+	// NOTE: This section is added to parse any fields in EnsemblingServiceK8sConfig that does not
+	// have yaml tags.
+	// For example `certificate-authority-data` is not unmarshalled
+	// by vipers unmarshal method.
```

Comment:
Since this block of code is performing a specific operation on a specific part of the `config` object (while the rest of the code in `Load` acts directly on `config` itself), would it be better to refactor this block into a separate helper function? 😅

Something like: 
```go
func Load(filepaths ...string) (*Config, error) {
	// ...
	config.ClusterConfig = loadClusterConfigs(v.AllSettings())
	return config, nil
}
```
It would also help make it easier to interpret why we're doing `return config, nil` prematurely whenever the `if !ok` statements return true.

Reply:
Yep that makes sense, i'll make the change